demo/chbench/mzcompose.yml

```diff
@@ -419,6 +437,10 @@ mzworkflows:
       seconds: 1
 
   download-snapshot:
+    env:
+      MZ_CHBENCH_SNAPSHOT: ${MZ_CHBENCH_SNAPSHOT:-'./tmp'}
+      MZ_CHBENCH_SNAPSHOT_SNAPSHOT: '${MZ_CHBENCH_SNAPSHOT:-./tmp}/kafka-data'
+      MZ_CHBENCH_SNAPSHOT_ZK_DATA: '${MZ_CHBENCH_SNAPSHOT:-./tmp}/zookeeper'
```

Comment:
I don't think download-snapshot requires these? It shouldn't be starting any of the Kafka / ZK / mzutil services.